Geometry/HcalCommonData/src/HcalDDDRecConstants.cc

```diff
@@ -2,12 +2,13 @@
 
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/Utilities/interface/Exception.h"
-
+#include "DataFormats/Math/interface/GeantUnits.h"
 #include "CLHEP/Units/GlobalPhysicalConstants.h"
 #include "CLHEP/Units/GlobalSystemOfUnits.h"
 #include <cmath>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```

Comment:
Are you sure you want debug code enabled all the time?

Reply:
No. But it won't create any printing right now in standard jobs. I shall fix this as well later